### DISCUSSION TITLE
Windows Flaresolverr installed cant find where to change environment variables

### DISCUSSION THREAD

**Discussion by Zuluking91** [FlareSolverr/FlareSolverr#1397] [Q&A]
As the titels says i cant seem to find where to change the  environment variables. Tried following this guide: https://www.rapidseedbox.com/blog/flaresolverr-guide but I somehow can't change anything. Can't type anything in CMD or powershell to change it.
![afbeelding](https://github.com/user-attachments/assets/458f7af1-1b96-453b-8674-bed183320ef0)

Setting up the proxy index in Prowlar I can see Flaresolverr is working but it gets caught up in Cloudflare protection CAPTCHA. So that needs to be changed to HCAPTCHA according to the guide. Any tips on how to get this done is greatly appreciated.

**Comment by Zuluking91** [comment]
Ive used docker instead and now its working the challenge but getting a http://localhost:8191/ internal server error 500 now.

**Comment by ilike2burnthing** [comment]
Ignore that guide, stick to our readme. To set ENVs - https://github.com/FlareSolverr/FlareSolverr#environment-variables

As for challenges not being successfully bypassed, for now use this contributor build from source:
https://github.com/21hsmw/FlareSolverr/tree/nodriver-support
or the Docker image:
21hsmw/flaresolverr:nodriver
